pkg/global/synapseconstants.go

```diff
@@ -34,13 +34,13 @@ var TASCloudURL map[string]string
 
 func init() {
 	SocketURL = map[string]string{
-		"stage": "wss://stage-api.tas.lambdatest.com/ws/",
+		"stage": "wss://stage-api-tas.lambdatestinternal.com/ws/",
 		"dev":   "ws://host.docker.internal/ws/",
 		"prod":  "wss://api.tas.lambdatest.com/ws/",
 		"pi":    "wss://api.tas-pi.lambdatest.com/ws/",
 	}
 	TASCloudURL = map[string]string{
-		"stage": "https://stage-api.tas.lambdatest.com",
+		"stage": "https://stage-api-tas.lambdatestinternal.com",
 		"dev":   "http://host.docker.internal",
 		"prod":  "https://api.tas.lambdatest.com",
 		"pi":    "https://api.tas-pi.lambdatest.com",
```

Comment:
@VikrantKS @nevilm-lt it should be read from env file. deployments shouldn't be present in code. Let's take this up ASAP.